config/labels/analyzers/gcc.json

```diff
@@ -0,0 +1,285 @@
+{
+  "analyzer": "gcc",
+  "labels": {
+    "-Wanalyzer-allocation-size": [
```

Comment:
These checker names should be exactly the same as what we provide for `--enable` flag, for example. These checker names should be listed by `CodeChecker checkers`. See Cppcheck, how we decided changing checker names compared to what Cppcheck provides itself.